ethcore/src/block.rs

```diff
@@ -558,7 +561,8 @@ fn enact(
 		parent,
 		last_hashes,
 		engine.executive_author(&header), // Engine such as Clique will calculate author from extra_data.  this is only important for executing contracts as the 'executive_author'
-		(*header.gas_limit(), *header.gas_limit()),
+		//(*header.gas_limit(), *header.gas_limit()),
+        (3141562.into(), 31415620.into()),
```

Comment:
I reset this in the hopes that it would magically solve the "block gas limit reached" problem we've encountered with syncing.. But it did not.